src/main/java/com/dmytrohuk/weborganizer/users/UserController.java

```diff
@@ -16,17 +17,22 @@ public UserController(UserService userService) {
     }
 
     @GetMapping
-    public List<User> getStudents(){
+    public List<User> getUsers(){
         return userService.getAllUsers();
     }
 
+    @GetMapping(path = "{userId}")
```

Comment:
better prepend slash to all path so it is something like `/{user-id}`.  Also all REST endpoint namings are in kebab-case (refer to https://restfulapi.net/resource-naming/)